doc/API.md

```diff
@@ -249,7 +249,7 @@ Runs a number of rules against the provided HTML page and returns the resulting
 #### Synopsis
 
 ```js
-axe.run(context, options, callback);
+const results = await axe.run(context, options);
```

Comment:
Many users were confused by top-level `await`s in our README of axe-puppeteer, which resulted in [this PR](https://github.com/dequelabs/axe-puppeteer/pull/14). If we're going to do this, we likely need to add similar documentation here.